### ISSUE TITLE
Make spec suffix configurable

### ISSUE THREAD

**Issue by jarekb84** [thekarel/vscode-open-spec-file#1]
Currently makeSpecFilePath.js uses a hardcoded suffix of '.spec', which limits usage of this extension to just that naming convention. If that suffix variable was exposed as a user configurable setting, then any number of test file naming conventions could be supported.

**Comment by jarekb84** [comment]
Some searching shows that you can add config properties to the contributes part of package.json

``` json
"configuration": {
  "type": "object",
  "title": "Open Spec File configuration",
  "properties": {
    "openSpecFile.specSuffix": {
      "type": "string",
      "default": ".spec",
      "description": "File suffix used to open or create a test file."
    }
  }
}
```

and then extract it via
```js
var config = vscode.workspace.getConfiguration('openSpecFile');
var SPEC_SUFFIX = config.specSuffix;
```


More info can be found [here](https://code.visualstudio.com/docs/extensionAPI/extension-points). If you want, I can give a shot at making a pull request for this, though I'd need to figure out how to test it.

**Comment by thekarel** [comment]
@jarekb84 can you give this a try? #2

**Comment by jarekb84** [comment]
@thekarel I haven't done any vscode extension development, so how would I try this locally? Pull down the git repo and then what?

**Comment by jarekb84** [comment]
Ok, got the branch working locally and with a few tweaks, got to setting to work as expected.

**Comment by jarekb84** [comment]
@thekarel I see you merged your branch and see the update in the marketplace. You should probably include a config section in your readme. Thanks for getting this done so quickly, it's a minor extension, but I really like the time saving/convenience it offers.

**Comment by thekarel** [comment]
Configuring the suffix is now available in version 0.0.3. Please do let me know if you have any more ideas for improvement!

**Comment by jarekb84** [comment]
Sure, this isn't something that I need, but the current logic assumes the src file and spec file live in the same directory, but plenty of projects keep them in totally seperate parts of the codebase. You could extend support to those use cases by adding config settings for src root and spec root. If both are set, when making the spec file, you can swap out the source root for spec root in the current file path when creating the specFileName. 

So in this scenario
```
src
   /foo
      /foo.js
tests
   /foo
      /foo-spec.js
```

When user hits the keyboard shortcut while in `./src/foo/foo.js` you'd open `./tests/foo/foo-spec.js` or create it. With the spec suffix feature you just added and this, this extension could support many other language file patterns then just javascript.
